src/components/archive/Archive.jsx

```diff
@@ -1,5 +1,39 @@
-function Archive() {
-  return <p>Archive</p>;
+import { BiTrash, IcOutlineUnarchive } from "assets/icons/Icons";
+import { useArchive } from "context";
+
+function Archive({ note }) {
+  const { title, color, description, label, priority } = note;
```

Comment:
suggestion - you can destructure the note prop in the argument itself, like this : `note : {title, color, description, label, priority}`

Reply:
yes, but i need to pass the note as an argument to function, so put it this way.